packages/ui/src/views/ContactsList.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <MainLayout>
+    <SectionHeader title="CONTACTS" />
+    <div v-if="player?.contacts" class="list-container">
+      <a
+        ref="downloadLink"
+        class="none"
+        :href="dataStr"
+        :download="'witty-creatures-contacts.json'"
+      ></a>
+      <input
+        ref="dataInput"
+        :style="{ display: 'none' }"
+        type="file"
+        @change="readFile"
+      />
+      <CustomButton type="primary" @click="exportContacts">
+        Export contact list
```

Comment:
Should we move this download logic to its own components?

Reply:
Implemented in #10